### ISSUE TITLE
Add support for Hybrid Renderer V2 (Unity 2020)

### ISSUE THREAD

**Issue by gabrjt** [vis2k/DOTSNET#9]
Although still in a preview stage, the Hybrid Renderer V2 module is a nice feature which brings great visual enhancements with great performance by supporting material property block.

As it is today, the Hybrid Renderer V2 does not supports being registered in more than one world.

Since this is a preview package that may be still changed a lot, maybe just a basic support will do.

What I've done so far is to assume that only the client will have the Hybrid Renderer Systems, so I've changed the Bootstrap in a hardcoded way to avoid registering the Hybrid Renderer Systems more than once.

```
Bootstrap.cs 

public static void CategorizeSystems(IReadOnlyList<Type> systems, out List<Type> unitySystems, out List<Type> regularSystems, out List<Type> serverSystems, out List<Type> clientSystems)

...
foreach (Type system in systems)
            {
                // [ServerWorld] and [ClientWorld]?
                if (SystemHasAttribute(system, typeof(ServerWorldAttribute)) &&
                    SystemHasAttribute(system, typeof(ClientWorldAttribute)))
                {
                    serverSystems.Add(system);
                    clientSystems.Add(system);
                }
                // [ServerWorld]?
                else if (SystemHasAttribute(system, typeof(ServerWorldAttribute)))
                {
                    serverSystems.Add(system);
                }
                // [ClientWorld]? Register the Rendering Systems
                else if (SystemHasAttribute(system, typeof(ClientWorldAttribute)) || system.Namespace == "Unity.Rendering")
                {
                    clientSystems.Add(system);
                }
                // no tag, in Unity namespace, and not a Rendering System?
                else if (system.Namespace != null && system.Namespace.StartsWith("Unity.") && system.Namespace != "Unity.Rendering")
                {
                    unitySystems.Add(system);
                }
                // otherwise it's a third party asset, or one of our systems
                // that doesn't have a [Client/ServerWorld] attribute
                else
                {
                    regularSystems.Add(system);
                }
            }
...

```

**Comment by vis2k** [comment]
looks like a good fix. will try.
did you hear anything about hybrid renderer being supported in multiple worlds later?

**Comment by gabrjt** [comment]
Unfortunately I'm not aware if there will be support for multiple worlds later. But the V2 is still in an early stage, so it may change in the future. 
While not mandatory, I think it's a good addition to the client side. Maybe even just writing in a documentation/wiki page how to enable V2 renderer instead of directly changing code in the package will do for now!

**Comment by vis2k** [comment]
@gabrjt thanks for the bug report. that was extremely useful.
fix will be in V1.14.
